packages/tools/client-debugger/client-debugger-view/src/components/ContainerSummaryView.tsx

```diff
@@ -141,35 +149,70 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 	return (
 		<Stack className="container-summary-view">
 			<StackItem>
-				<span>
-					<b>Container</b>:{" "}
-					{containerState.nickname !== undefined
-						? `${containerState.nickname} (${containerState.id})`
-						: containerState.id}
-				</span>
-			</StackItem>
-			<StackItem>
-				<span>
-					<b>Status</b>: {statusString}
-				</span>
-			</StackItem>
-			<StackItem>
-				{containerState.clientId === undefined ? (
-					<></>
-				) : (
-					<span>
-						<b>Client ID</b>: {containerState.clientId}
-					</span>
-				)}
-			</StackItem>
-			<StackItem>
-				{containerState.audienceId === undefined ? (
-					<></>
-				) : (
-					<span>
-						<b>Audience ID</b>: {containerState.audienceId}
-					</span>
-				)}
+				<Table>
+					<TableBody>
+						<TableRow>
+							<TableCell>
+								<TableCellLayout>
+									<b>Container</b>:
+								</TableCellLayout>
+							</TableCell>
+							<TableCell>
+								<TableCellLayout>
+									{containerState.nickname !== undefined
+										? `${containerState.nickname} (${containerState.id})`
+										: containerState.id}
+								</TableCellLayout>
+							</TableCell>
+						</TableRow>
+						<TableRow>
+							<TableCell>
+								<TableCellLayout>
+									<b>Status</b>:
+								</TableCellLayout>
+							</TableCell>
+							<TableCell>
+								<TableCellLayout>
+									{((): JSX.Element => {
+										switch (statusString) {
+											case "disconnected":
+												return <Badge color="danger">{statusString}</Badge>;
+											case "detached":
+												return (
+													<Badge color="warning">{statusString}</Badge>
+												);
+											default:
+												return (
+													<Badge color="success">{statusString}</Badge>
+												);
+										}
+									})()}
+								</TableCellLayout>
+							</TableCell>
+						</TableRow>
+						<TableRow>
+							<TableCell>
+								<TableCellLayout>
+									<b>Client ID</b>:
```

Comment:
Styling nits: 
- Can we get rid of the `:` as they are not table columns?
- We should use the default width propert of the table and reduce the width of the first column to allow more space for 2nd column which has longer information
- I would recommend using size `small` or `extra small` (this is what I am leaning towards) as dev tools typically match that style (matching other browser dev tools)